packages/api/test/functional/technology.spec.js

```diff
@@ -318,6 +318,43 @@ test('POST /technologies creates/saves a new technology.', async ({ client, asse
 	response.assertJSONSubset(technologyCreated.toJSON());
 });
 
+test('POST /technologies creates/saves a new technology and check slug', async ({
```

Comment:
Vamos mudar o título deste teste para ficar mais claro:

Algo como: "technology slug is not created with unwanted characters"